lib/utils/compose.js

```diff
@@ -641,8 +365,7 @@ export const pushAndUpdateServiceImages = function (
 		Promise.all(
 			images.map(({ serviceImage, localImage, props, logs }, index) =>
 				Promise.all([
-					// @ts-ignore
-					localImage.inspect().get('Size'),
+					localImage.inspect().then((img) => img.Size),
```

Comment:
Typing info for `docker-toolbelt` declares native promises, but apparently it actually uses Bluebird promises and the old CLI code was relying on Bluebird being used -- the `.get('Size')` call was Bluebird's `.get`:
http://bluebirdjs.com/docs/api/get.html
This type _vs._ usage incompatibility was ignored with a `"// @ts-ignore"` comment. I've changed the code here and elsewhere not to assume Bluebird, and removed the `ts-ignore` comment.